docker-compose.yaml

```diff
@@ -26,15 +19,16 @@ services:
     environment:
       DISCORD_TOKEN: ${DISCORD_TOKEN}
       OCTOKIT_TOKEN: ${OCTOKIT_TOKEN}
-      MONGODB_URL: mongodb://mongo:27017/tournamentdb
+      MONGODB_URL: UNUSED
       CHALLONGE_USERNAME: ${CHALLONGE_USERNAME}
       CHALLONGE_TOKEN: ${CHALLONGE_TOKEN}
       POSTGRESQL_URL: "postgresql://${POSTGRES_USER}:${POSTGRES_PASSWORD}@postgres/${POSTGRES_DB}"
+      EMCEE_USE_POSTGRES: 1
```

Comment:
Do we need to include `NODE_ENVIRONMENT` like we had issues with last night, or does the different structure take care of that?

Reply:
It's not needed. synchronize is forced to true until further notice